storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -139,6 +139,9 @@
     @isString
     public static final String STORM_META_SERIALIZATION_DELEGATE = "storm.meta.serialization.delegate";
 
+    @isType(type=List.class)
```

Comment:
My question was mostly around can we make the config checks more specific instead of just a list of who knows what.

And at a minimum have some documentation about what should be stored in this config.